api/pkg/cli/hub/get_catalog.go

```diff
@@ -0,0 +1,76 @@
+package hub
+
+import (
+	"encoding/json"
+	"net/http"
+
+	cclient "github.com/tektoncd/hub/api/v1/gen/http/catalog/client"
+)
+
+type CatalogOptions struct {
+}
+
+type CatalogResult struct {
+	data    []byte
+	status  int
+	err     error
+	Catalog CatalogData
+}
+type CatalogData = cclient.ListResponseBody
+
+func (c *client) GetAllCatalogs() CatalogResult {
+	data, status, err := c.Get(catalogEndpoint())
+	if status == http.StatusNotFound {
```

Comment:
Api doesn't return 404 currently if there are not catalogs in db... i think it should 🤔